### ISSUE TITLE
Arrow keys don’t change singer

### ISSUE THREAD

**Issue by backstagekaraoke** [OpenKJ/OpenKJ#113]
Arrow keys dont change singer proper
 - when using the arrow keys on keyboard to cycle through the singers list, it does change the highlighted singer
 - however it does not change to that singers song queue, it stays on whichever one was selected with the mouse
 - only clicking on the singer with the mouse changes to display their song queue

Mac OS 10.13 High Sierra
OpenKJ 1.4.5

**Comment by coyote1357** [comment]
Good catch.  I'll change the behaviour to on selection changed instead of just on click.

**Comment by backstagekaraoke** [comment]
awesome 👍

**Comment by coyote1357** [comment]
Fixed in 1.5.18 unstable
